cmd/samples/samples.go

```diff
@@ -31,12 +32,28 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	supportedTemplates = map[string]struct{}{
```

Comment:
This map might be a good way to map the the template name to the template directory so you can simply look it up from here.